### DISCUSSION TITLE
How to set a Pump characteristics

### DISCUSSION THREAD

**Discussion by ZackeyZAKI** [OpenModelica/OpenModelica#10735] [Q&A]
When entering pump data in OMEdit, the following warning is displayed:
Is there any way to deal with it without having to input the head-flow characteristic?

[Modelica.Fluid.Machines: 328:5-329:76]: [The following equation is INCONSISTENT due to specified unit information: pump.delta_head_init = Package_test.XXX.pump.flowCharacteristic(pump.V_flow_single_init * 1.1, {0.0, 0.025, 0.05}, {10.0, 6.0, 0.0}) - Package_test.XXX.pump.flowCharacteristic(pump.V_flow_single_init, {0.0, 0.025, 0.05}, {10.0, 6.0, 0.0})](omeditmessagesbrowser:///Modelica.Fluid.Machines?lineNumber=328)

The units of following sub-expressions need to be equal:
- sub-expression "pump.V_flow_single_init * 1.1" has unit "m"
- sub-expression "Package_test.XXX.pump.flowCharacteristic().V_flow" has unit "m3/s"

**Comment by casella** [comment]
This warning is a bit strange, we need to investigate it. I think you can ignore it. However, you should definitely set up a test model where you run the pump with known boundary conditions and rotational speed, and compare the simulation results with what you'd expect from hand computations based on the supplied curves. The result should match, at least as a first approximation.
